test/libsolidity/smtCheckerTests/operators/bitwise_xor_int.sol

```diff
@@ -0,0 +1,19 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f() public pure {
+		int8 x = 1;
+		int16 y = 0;
+		assert(x ^ y == 1);
+		int16 z = -1;
+		assert(x ^ z == -2);
+		assert(y ^ z == -1);
+		assert(y ^ z > 0);
+		x = 7; y = 3;
+		assert(x ^ y < 5);
+		assert(x ^ y > 5);
+	}
+}
+// ----
+// Warning 6328: (189-206): Assertion violation happens here
+// Warning 6328: (247-264): Assertion violation happens here
```

Comment:
Now I get what @ekpyron goes through when reviewing these PRs, I have no idea which assertions are actually failing :p

Reply:
Yes, I can imagine how many of these he have checked :)